extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -225,7 +246,8 @@ function sendRtcCallout_(
             return {rtcTime, callout};
```

Comment:
should `callout` here be trimmed?

Reply:
Changed where we do the trimming so we only need to do it once.